save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/testsuite/TestSuitesSourceDto.kt

```diff
@@ -24,6 +25,7 @@ data class TestSuitesSourceDto(
     val gitDto: GitDto,
     val branch: String,
     val testRootPath: String,
+    val latestVersion: String = "",
```

Comment:
If it is possible to have an empty value here, maybe better make it nullable? Or if empty value is not possible, it's better to remove the default value

Reply:
Empty value is possible as far as we have created test suite source, but not fetched yet. I see no difference whether to check nullability or emptiness

Reply:
For me nullable field says more explicitly that the value can be missing. To handle empty strings as an empty value you have to always keep it in mind, while nullability leads to some enforcement during compilation.

Reply:
Makes sense, will fix it